src/components/atoms/ChefCard/ChefCard.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image"
+import Link from "next/link"
+
+type ChefCardProps = {
+  icon: string
+  name: string
+  text: string
+  args: string
+  count: number
```

Comment:
propsの命名わかりやすくしましょう〜！
以下、案です 👍 
```suggestion
  image: string
  name: string
  profile: string
  url: string
  numberOfRecipes: number
```

Reply:
ありがとうございます！
修正案ありがとうございます。
頂きました内容に修正させて頂きました✨🐱